app/javascript/components/api/boards-api.js

```diff
@@ -2,85 +2,36 @@ import {api} from './api';
 
 export const boardApi = {
   async getBoards() {
-    try {
-      const {data} = await api.get(`boards/my`);
-      return data;
-    } catch (error) {
-      if (error.response) {
-        alert(error.response.data.errors.fullMessages);
-      } else {
-        alert('Something went wrong, try again.');
-      }
-    }
+    const {data} = await api.get(`boards/my`);
+    return data;
   },
   async getBoardsWhereIAm() {
-    try {
-      const {data} = await api.get(`boards/participating`);
-      return data;
-    } catch (error) {
-      if (error.response) {
-        alert(error.response.data.errors.fullMessages);
-      } else {
-        alert('Something went wrong, try again.');
-      }
-    }
+    const {data} = await api.get(`boards/participating`);
+    return data;
   },
   async getBoard(boardSlug) {
```

Comment:
This request is wrong and isn't used, so please remove or correct it
Response is the same `{ data: BOARD_STRUCT }`
```
const {data} = await api.get(`boards/${boardSlug}`);
return data;
```